framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java

```diff
@@ -71,6 +71,10 @@ public void setOpenShift(OpenShift openShift) {
         this.openShift = openShift;
     }
 
+    protected Project getProject() {
```

Comment:
I think this is a leftover. If we remove the changes in HttpsWorkbenchTestProvider, this is not used anywhere.